Testing glXCreateContextAttribsARB() isn't really necessary and creates build
issues because the function isn't available in all OpenGL implementations.
--- a/rr/rrfakerut.cpp
+++ b/rr/rrfakerut.cpp
@@ -1173,10 +1173,9 @@
 		getcfgattrib(c, GLX_RENDER_TYPE, render_type);
 		render_type=(render_type==GLX_COLOR_INDEX_BIT)? GLX_COLOR_INDEX_TYPE:GLX_RGBA_TYPE;
 		fbcid=cfgid(dpy, c);
-		int attribs[]={GLX_RENDER_TYPE, render_type, 0};
 
-		if(!(ctx=glXCreateContextAttribsARB(dpy, c, NULL, True, attribs)))
-			_error("glXCreateContextAttribsARB");
+		if(!(ctx=glXCreateNewContext(dpy, c, render_type, NULL, True)))
+			_error("glXCreateNewContext");
 		temp=-20;
 		glXQueryContext(dpy, ctx, GLX_FBCONFIG_ID, &temp);
 		if(temp!=fbcid) _error("glXQueryContext FB cfg ID");
